src/front/shared/pages/Exchange/QuickSwap/index.tsx

```diff
@@ -774,8 +807,17 @@ class QuickSwap extends PureComponent<IUniversalObj, ComponentState> {
       <>
         <NewTokenInstruction />
 
+        {receivedCurrency.notExist && (
+          <p styleName="noAssetsNotice">
+            <FormattedMessage
+              id="notEnoughAssetsNotice"
+              defaultMessage="You do not have enough assets in the wallet. Please add more of them to be able to swap."
```

Comment:
You don't have available assets in this network to exchange. Please change the network or add a custom asset to the wallet